client-src/elements/chromedash-roadmap-milestone-card.ts

```diff
@@ -249,6 +287,17 @@ class ChromedashRoadmapMilestoneCard extends LitElement {
           ${f.name}
         </a>
         <span class="icon_row">
+          ${this._isFeatureOutdated(f.accurate_as_of, this.channel?.version)
+            ? html`
+                <span
+                  class="tooltip"
+                  id="oudated-icon"
```

Comment:
Oops:

```suggestion
                  id="outdated-icon"
```

And in all the tests.

Reply:
oops thanks!